packages/metadata/src/__tests__/unit/decorator-factory.unit.ts

```diff
@@ -113,7 +113,7 @@ describe('ClassDecoratorFactory', () => {
     expect(() => {
       @classDecorator({x: 1})
       @classDecorator({y: 2})
-      // tslint:disable-next-line:no-unused
+      /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Please keep using single-line comments, I believe eslint does support them.

```suggestion
      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Same comment applies to all other places your are changing in this pull request.

Reply:
I noticed that `eslint` only accepts `/* eslint-disable ...  */. 

`// eslint-disable-next-line ...` seems to be fine.

Reply:
Interesting. 

Anyhow, since `// eslint-disable-next-line` is supported according to your analysis, I think we should keep using it.

I am ok to use `/*` for overrides that apply to more than one line, if that's what eslint forces us to do. I actually quite like that on the second thought, because it will make such statement to stand out from our regular code and make them easier to spot during code reviews.